Day-01/typescript/zFlxw/day01.ts

```diff
@@ -0,0 +1,53 @@
+// Made using the Deno.land runtime. The utils.ts file can be found in /shared/zFlxw.
+
+import { readFileLines } from '../../../shared/zFlxw/utils.ts';
+
+export function day01_01() {
+  const lines = readFileLines('day01/input.txt');
```

Comment:
Wusste ich doch, dass ich zwei mal die Datei gesehen habe... Sorry, war irgendwie blind

```suggestion
  const lines = readFileLines('input.txt');
```